testsuite/classlibrary/TestSoundFile.sc

```diff
@@ -0,0 +1,51 @@
+TestSoundFile : UnitTest {
+
+	var server, soundFile, path, info;
+
+	setUp {
+
+		path = Platform.resourceDir +/+ "sounds/a11wlk01.wav";
+		info = IdentityDictionary[
+			(\numFrames -> 188893),
+			(\numChannels -> 1),
+			(\path -> path),
+			(\sampleRate -> 44100),
+			(\sampleFormat -> "int16"),
+			(\headerFormat -> "WAV")
+		];
+		soundFile = SoundFile(path);
+
+	}
+
+	tearDown {
+
+		soundFile.close;
+
+	}
+
+	test_isOpen {
+
+		soundFile.openRead;
+		this.assert(soundFile.isOpen, "SoundFile should now be opened");
+
+		soundFile.close;
+		this.assert(soundFile.isOpen.not, "SoundFile should now be closed");
+
+	}
```

Comment:
ideally this would be two separate tests - the first assert checks that `openRead` and `isOpen` work together; the second assert checks that `close` and `isOpen` work together

Reply:
not a huge deal, but something to keep in mind for future tests.